modules/standard/AutoMath.chpl

```diff
@@ -865,13 +865,37 @@ module AutoMath {
      `false` otherwise. */
   inline proc isnan(x: real(32)): bool do return chpl_macro_float_isnan(x):bool;
 
-  /* Multiply by an integer power of 2.
-     Returns x * 2**n.
-     */
-  pragma "fn synchronization free"
-  pragma "codegen for CPU and GPU"
-  extern proc ldexp(x:real(64), n:int(32)):real(64);
+  // When removing this deprecated function, be sure to remove chpl_ldexp and
+  // move its contents into Math.chpl to reduce the symbols living in this
+  // module.
+  pragma "last resort"
+  @deprecated(notes="In an upcoming release 'ldexp' will no longer be included by default, please 'use' or 'import' the 'Math' module to call it")
```

Comment:
I would consider writing 'Math' as ':mod:\`Math\`', but its a tiny doc improvement so up to you